src/Hit/Git/Stash.hs

```diff
@@ -11,4 +11,4 @@ import Shellmet()
 runStash :: IO ()
 runStash = do
     "git" ["add", "."]
-    "git" ["stash"]
+    "git" ["stash", "--include-untracked"]
```

Comment:
After adding this option you can remove `git add .`. So now this will be a single command instead of two commands, that was the motivation behind this issue.